apps/notifications/src/lib/bootstrap/bootstrap.spec.ts

```
@@ -0,0 +1,68 @@
+      expect.any(HttpResultInterceptor),
+    );
+    expect(mockApp.useGlobalFilters).toHaveBeenCalledWith(
+      expect.any(HttpExceptionFilter),
+    );
+
+    expect(mockApp.listen).toHaveBeenCalledWith('4321');
+    expect(mockLogger.log).toHaveBeenCalledWith('Server started on port 4321');
+  });
+
+  it('should fallback to port 3000 if PORT is not set', async () => {
+    delete process.env.PORT;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)